src/Config.js

```diff
@@ -0,0 +1,3 @@
+export const Config = {
+    apiHost: '/api'
+}
```

Comment:
If you want to keep it private you can place it in the env file itself and then use it anywhere by using process.env.apiHost and similarly on Netlify you can pass this env variable for the live previews to work accordingly!!

Reply:
This is not for private purpose. Just wanted to abstract out the entire prefix url of the API in a separate file as it could vary from API to API.